src/DistributionsAD.jl

```diff
@@ -49,14 +43,39 @@ export TuringScalMvNormal,
 
 include("common.jl")
 include("univariate.jl")
+#=
 include("multivariate.jl")
 include("mvcategorical.jl")
 include("matrixvariate.jl")
 include("flatten.jl")
 include("arraydist.jl")
 include("filldist.jl")
-@init @require ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267" begin
-    include("reversediff.jl")
+=#
+function __init__()
+    @require ForwardDiff="f6369f11-7733-5829-9624-2563aa707210" begin
+        using ForwardDiff: @define_binary_dual_op # Needed for `eval`ing diffrules here
+        include("forwarddiff.jl")
+    end
+
+    @require ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267" begin
+        include("reversediff.jl")
+    end
+
+    @require Tracker="9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c" begin
+        using DiffRules
+        using ZygoteRules
+        using SpecialFunctions
+        using LinearAlgebra: AbstractTriangular
+        using Tracker: Tracker, TrackedReal, TrackedVector, TrackedMatrix,
```

Comment:
Requires requires to load the optional dependencies with a `.` in front of the name (see https://github.com/JuliaPackaging/Requires.jl):
```suggestion
        using .Tracker: Tracker, TrackedReal, TrackedVector, TrackedMatrix,
```
Otherwise users will get a lot of warnings.